cmd/common-main.go

```diff
@@ -123,3 +123,23 @@ func handleCommonEnvVars() {
 		globalIsEnvDomainName = true
 	}
 }
+
+func loadCerts() {
+	// Check for certificates only if endpoints are secure.
+	if ssl := globalEndpoints.IsHTTPS(); ssl {
```

Comment:
actually `globalIsSSL = globalEndpoints.IsHTTPS()`.  If possible `globalIsSSL` can be removed.

Reply:
No it's not true for gateway or FS or standalone Erasure.